com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -532,8 +538,11 @@ public void build(Task<Void> task) throws CompileExceptionError, IOException {
                 HashSet<String> resources = findResources(project, rootNode);
 
                 List<String> excludedResources = new ArrayList<String>();
-                for (String excludedResource : project.getExcludedCollectionProxies()) {
-                    excludedResources.add(excludedResource);
+                boolean shouldPublish = project.option("liveupdate", "false").equals("true");
```

Comment:
even if there are some collections in excludes list, we should take it into account only if `--liveupdate` is `true`